packages/zero-client/src/client/custom.test.ts

```diff
@@ -128,3 +135,158 @@ test('custom mutators write to the local store', async () => {
   issues = await z.query.issue.run();
   expect(issues.length).toEqual(0);
 });
+
+test('custom mutators can query the local store during an optimistic mutation', async () => {
+  const z = zeroForTest({
+    schema,
+    mutators: {
+      issue: {
+        create: async (tx, args: InsertValue<typeof schema.tables.issue>) => {
```

Comment:
OT, does the following work now:

```js
mutators: doSomething(tx) { ... {
```

or is the 2 level definition still required?

Reply:
It doesn't. I'll fix that today.